src/gui/src/renderThread.cpp

```diff
@@ -697,13 +700,23 @@ bool RenderThread::drawTextInBBox(const QColor& text_color,
   painter->scale(scale_adjust, -scale_adjust);
   if (do_rotate) {
     text_bounding_box = font_metrics.boundingRect(name);
+    auto xOffset = (bbox_in_px.height() - text_bounding_box.width()) / 2;
+    auto yOffset = (bbox_in_px.width() - text_bounding_box.height()) / 2;
```

Comment:
Move inside 'if (center)' since they are only needed at that scope.  Also const.  Same in the else block.